src/main/webapp/app/components/sidebar/navigation-sidebar.scss

```diff
@@ -1,20 +1,124 @@
 @import '../../variables.scss';
 
-$sidebar-bg-color: whitesmoke;
-$sidebar-color: $oncokb-blue;
-$submenu-bg-color: whitesmoke;
-$sidebar-width: 200px;
-$sidebar-collapsed-wdith: 80px;
-$highlight-color: $oncokb-darker-blue;
+.ps-menu-divider {
```

Comment:
`ps` is?

Reply:
`ps` is the prefix that the ProSidebar library uses for their classnames. 

Since this is my own class, I changed it to `nav-menu-divider`.